velox/connectors/hive/iceberg/tests/IcebergReadTest.cpp

```diff
@@ -225,6 +226,41 @@ class HiveIcebergTest : public HiveConnectorTestBase {
     ASSERT_TRUE(it->second.peakMemoryBytes > 0);
   }
 
+  void assertQuery(
```

Comment:
Maybe rename this function to assertPartitionKey.

Reply:
Want to keep this name generic, since it could be used to test any case.

Reply:
This method is building a very specific plan with TableScanNode and IcebergSplits... The assertQuery function name is being used in all the TestBase classes for very generic usage.

Reply:
assertPartitionKey also doesn't seem to be the right name, since we can just pass empty partition key. It is just another overloaded assertQuery method.